doc/radosgw/uadk-accel.rst

```diff
@@ -33,10 +33,10 @@ See `OpenSSL UADK Engine`_.
 
 UADK Environment Setup
 ======================
-UADK consists of UACCE, vendors’ drivers, and an algorithm layer. UADK requires the
-hardware accelerator to support SVA, and the operating system to support IOMMU and
-SVA. Hardware accelerators from different vendors are registered as different character
-devices with UACCE by using kernel-mode drivers of the vendors.
+UADK consists of UACCE, vendor drivers, and an algorithm layer. UADK requires
+the hardware accelerator to support SVA, and the operating system to support
+IOMMU and SVA. Hardware accelerators are registered as different character
```

Comment:
Recent wisdom is to disable IOMMU for performance on AMD EPYC systems, c.f. Mark Nelson.  This would seem to imply that this acceleration framework is incompatible with that strategy, so I would like to see consultation with someone who understands these bits wrt whether this is true, and if so if we should call it out here and suggest which if either should win.

Reply:
IOMMU configuration could be partially set to bypass mode. So in theory, both improvement gains could be achieved. Would have a test later.